docs/how-to/h-manage-service-accounts.md

```diff
@@ -1,72 +1,89 @@
 ## Manage Spark Client Accounts
 
-This section provides general commands on how to use the CLI interface for creating, managing and configuring spark applications.
+This is an introduction to the CLI interface for creating, managing and configuring Spark accounts.
 
 ### Create Service Account
 
+Note: In case using another namespace than `default`, make sure that it already exists in Kubernetes.
 ```bash
-spark-client.service-account-registry --username demouser --namespace demonamespace create --primary --properties-file /home/demouser/conf/spark-overrides.conf  --conf spark.app.name=demo-spark-app-overrides
+$ kubectl create namespace demonamespace
+namespace/demonamespace created
 ```
 
-This creates a service account associated with configuration properties provided either via property file or explicit 
-configuration arguments. The flags `--primary` specifies that the newly create account will be the primary account to 
-be used. If another primary exists, the latter account primary flag will be set to `false`.
+Now we can define a service account within the scope of this namespace. In case we may have a configuration file with default settings for the account, we could also include those when defining the account. The syntax of the config file is identical to [Spark Configuration Properties](https://spark.apache.org/docs/latest/configuration.html#available-properties). For example
+```bash
+$ echo "spark.kubernetes.deploy-mode=cluster" > /home/demouser/conf/spark-overrides.conf
+```
 
-### List all service accounts
+The command below creates a service account associated with configuration properties provided either via property file or explicit 
+configuration arguments.  The flags `--primary` specifies that the newly created account will be the primary account to 
+be used. (If another primary exists, the latter account primary flag will be set to `false`.)
 
 ```bash
-spark-client.service-account-registry list
+spark-client.service-account-registry create --username demouser --namespace demonamespace  --primary --properties-file /home/demouser/conf/spark-overrides.conf  --conf spark.app.name=demo-spark-app-overrides
```

Comment:
This command would fail actually, with an exception. However, the output is not too clear and it is indeed a rather common use-caes and exception that should be handled appropriately. I have raised an issue to be addressed in the next pulses. When addressing this, we will also update the documentation [here](https://github.com/canonical/spark-k8s-toolkit-py/issues/23)